ui/app/build-configs/buildTypes.js

```diff
@@ -0,0 +1,36 @@
+/*
+ * JBoss, Home of Professional Open Source.
+ * Copyright 2014-2019 Red Hat, Inc., and individual contributors
+ * as indicated by the @author tags.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+(function () {
+  'use strict';
+
+  angular.module('pnc.build-configs').factory('buildTypes', [
+    function () {
+      return [
+        {
+          id: 'MVN',
+          name: 'Maven'
+        },
+        {
+          id: 'NPM',
```

Comment:
@alexcreasy Looks good! I would remove here NPM since it's due for 2.0, and add the content of PR (to be merged in master: https://github.com/project-ncl/pnc/pull/2400/files):
id: 'GRADLE',
name: 'Gradle'

Reply:
Good idea, changes added!